README.md

```diff
@@ -2,18 +2,20 @@
 This is the POC of Jenkinsfile Runner Action for GitHub Actions in GSoC 2022.
 
 ## Introduction
-Jenkinsfile Runner Action for GitHub Actions aims at providing one-time runtime context for Jenkins pipeline. The users are able to run the pipeline at the GitHub Actions by only providing the Jenkinsfile and the definition of GitHub workflow. This project is powered by [jenkinsfile-runner](https://github.com/jenkinsci/jenkinsfile-runner). 
+Jenkinsfile Runner Action for GitHub Actions aims at providing one-time runtime context for Jenkins pipeline. The users are able to run the pipeline at the GitHub Actions by only providing the Jenkinsfile and the definition of GitHub workflow. This project is powered by [jenkinsfile-runner](https://github.com/jenkinsci/jenkinsfile-runner) mainly. The plugin downloading step is powered by [plugin-installation-manager-tool](https://github.com/jenkinsci/plugin-installation-manager-tool).
 
 You can configure the pipeline environment by using other GitHub Actions or providing JCasC Yaml file powered by [configuration-as-code-plugin](https://www.jenkins.io/projects/jcasc/).
 
+This project is only a simple user case of Jenkinsfile-runner which uses GitHub Actions as a FaaS context. If the users have such a need, they can deploy it to any FaaS contexts for one time job such as AWS Function and Azure Function.
```

Comment:
```suggestion
This project is only a simple use case of Jenkinsfile-runner which uses GitHub Actions as a FaaS context. If the users have such a need, they can deploy it to any FaaS contexts for one time job such as AWS Function and Azure Function.
```

I find this paragraph confusing. How are Azure functions related to this action?

Reply:
You might want to see [the presentation](https://docs.google.com/presentation/d/1y7YnAdnh5WY59g8oIGTsj8sLQ5KXgoV7uUCBkxcTU88/edit#slide=id.p1) given by Oleg. This project is only a simple use case.

Reply:
Why do we call these "AWS Function" and "Azure Function"? Do you mean "Serverless Functions"? Maybe you could define what these terms properly and cite some existing examples like "AWS Lambda" and "Azure Functions"? (Say describe briefly what they are and do as well as how they are used.) How about similar features from GCP and IBM? We should try to be as comprehensive as possible.

Reply:
I just deleted this vague paragraph in the introduction. Sorry for the misunderstanding.